### PR TITLE
[WIP] yung caption changes

### PR DESCRIPTION
**Pull request by yungtingchen**
# caption changes

## caption

### CODE REVIEW

**Review by dcranfill — APPROVED** [review]
Code review complete!

File: _posts/0000-01-19-yungtingchen.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
+﻿---
 layout: slide
 title: "Welcome to our slide deck!"
 ---
 
 ![humpday](https://cloud.githubusercontent.com/assets/16547949/25400910/10641d9e-29c2-11e7-80c8-5c38ea261b22.jpg)
 {: .center}
 
-Caption here
+!!!!!!updated
```
Comment (dcranfill) [inline _posts/0000-01-19-yungtingchen.md:9]: 😊

Comment (yungtingchen) [reply _posts/0000-01-19-yungtingchen.md:9]: Thanks!

File: _posts/0000-01-19-yungtingchen.md [hunk 2]
```diff
@@ -1,9 +1,10 @@
...
+another change
```
Comment (dcranfill) [inline _posts/0000-01-19-yungtingchen.md:10]: 😄 Good to go!

**Pull request merged**